### ISSUE TITLE
"3.1-ubuntu" tag doesn't exist

### ISSUE THREAD

**Issue by fitz123** [jrottenberg/ffmpeg#17]
Something is wrong with tags description

```
$ docker pull jrottenberg/ffmpeg:3.1-ubuntu
Pulling repository docker.io/jrottenberg/ffmpeg
Tag 3.1-ubuntu not found in repository docker.io/jrottenberg/ffmpeg
```

**Comment by jrottenberg** [comment]
Ah my bad, let me update the wiki it's misleading, 

- 3 
- 3.1
- ubuntu

are different tags from the same image.

That should be fix with #15

**Comment by jrottenberg** [comment]
Had to reorganize the tagging a bit but it should be more consistent and simple going forward.
